src/overlays/actors/ovl_Dm_Char02/z_dm_char02.c

```diff
@@ -31,147 +31,147 @@ const ActorInit Dm_Char02_InitVars = {
 
 };
 
-static AnimationInfo sAnimations[4] = {
+static AnimationInfo sAnimations[] = {
     { &object_stk2_Anim_00A2BC, 1.0f, 0.0f, -1.0f, 2, 0.0f },
     { &object_stk2_Anim_00A4C4, 1.0f, 0.0f, -1.0f, 2, 0.0f },
     { &object_stk2_Anim_00A6D4, 1.0f, 0.0f, -1.0f, 0, 0.0f },
     { &object_stk2_Anim_00A104, 1.0f, 0.0f, -1.0f, 2, 0.0f },
 };
 
-void DmChar02_ChangeAnimationByInfo(SkelAnime* skelAnime, AnimationInfo* animation, u16 idx) {
+void DmChar02_ChangeAnimationByInfo(SkelAnime* skelAnime, AnimationInfo* animInfo, u16 idx) {
```

Comment:
One other thing here is that we are replacing `idx` by `index` for the sake of clarity. You don't have to do it now, it'll get cleaned up soon anyway.

Reply:
might as well :woman_shrugging: